job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -30,17 +30,17 @@ def KOGITO_BLUE_NEXT_PRODUCT_VERSION='1.13.2.blue'
 def KOGITO_BLUE_NEXT_PRODUCT_BRANCH='1.13.x-blue'
 def KOGITO_BLUE_NEXT_PRODUCT_CONFIG_BRANCH="kogito/1.13.x-blue"
 
-def SERVERLESS_LOGIC_CURRENT_PRODUCT_VERSION='2.0.0'
-def SERVERLESS_LOGIC_KOGITO_CURRENT_PRODUCT_VERSION='2.0.0'
-def SERVERLESS_LOGIC_DROOLS_CURRENT_PRODUCT_VERSION='8.26.0'
-def SERVERLESS_LOGIC_CURRENT_PRODUCT_BRANCH='main'
-def SERVERLESS_LOGIC_CURRENT_PRODUCT_CONFIG_BRANCH="master"
-
-def SERVERLESS_LOGIC_NEXT_PRODUCT_VERSION='1.25.0'
-def SERVERLESS_LOGIC_KOGITO_NEXT_PRODUCT_VERSION='1.27.0'
-def SERVERLESS_LOGIC_DROOLS_NEXT_PRODUCT_VERSION='8.27.0'
-def SERVERLESS_LOGIC_NEXT_PRODUCT_BRANCH='1.27.x'
-def SERVERLESS_LOGIC_NEXT_PRODUCT_CONFIG_BRANCH="openshift-serverless-logic/1.27.x"
+def SERVERLESS_LOGIC_NEXT_PRODUCT_VERSION='2.0.0'
+def SERVERLESS_LOGIC_KOGITO_NEXT_PRODUCT_VERSION='2.0.0'
+def SERVERLESS_LOGIC_DROOLS_NEXT_PRODUCT_VERSION='8.28.0'
```

Comment:
@lampajr We could ask the `prepare-release-branch` from Kogito project to update that value automatically after creating the branch. what do you think ?
That way, you don't need to keep up on every community release.

Reply:
@radtriste I definetely agree with you